types/index.d.ts

```diff
@@ -0,0 +1,273 @@
+import type { TestRunner } from "atom"
+//@ts-ignore
+import type * as Jasmine from "jasmine"
```

Comment:
We should be able to remove the `//@ts-ignore` if we add `esModuleInterop` to `tsconfig.json`

Reply:
That works here, but not when other people are using this package as a dependency. So, it is better to `@ts-ignore` it at the source of the issue instead of asking all the users to mute this error.